### ISSUE TITLE
Inception Score and SSIM score on market-1501 of test

### ISSUE THREAD

**Issue by longintlong** [tengteng95/Pose-Transfer#31]
Hi! Thanks for your open-source code of your project.  I have few questions about IS and SSIM scores
I followed the readme, using your data and code ,but i got the different result  from your paper:
IS 3.1618128 
Mask-IS 3.735669 
SSIM 0.281508094  
Mask-SSIM 0.799356671
what is the reason?

**Comment by jessemelpolio** [comment]
Which dataset do you refer to? Deepfashion or Market?

**Comment by longintlong** [comment]
I used market data

**Comment by tengteng95** [comment]
Hi, could you show me the details of your testing environment?

**Comment by longintlong** [comment]
> Hi, could you show me the details of your testing environment?

I used pytorch1.1 and tensorflow-gpu 1.4

**Comment by tengteng95** [comment]
We test our model on both Pytorch0.3.1 and Pytorch1.0.1, and the results are consistent. Have you tried Pytorch0.3.1?

**Comment by longintlong** [comment]
> We test our model on both Pytorch0.3.1 and Pytorch1.0.1, and the results are consistent. Have you tried Pytorch0.3.1?

Could you give me the code for pytorch0.3.1? Thank you ~
